styles/main.css

```diff
@@ -272,6 +273,7 @@ img {
         border-top: 5px solid black;
         margin-top: 1em;
         cursor: pointer;
+        position: relative;

```

Comment:
You need to give the .burger element a non-static position so that the absolutely-positioned ::before and ::after elements are placed relative to it